sdk/core/Azure.Core.Amqp/src/AmqpAddress.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Core.Amqp
+{
+    /// <summary>
+    /// Represents the AMQP address.
+    /// <seealso href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-address-string"/>.
```

Comment:
Thank you for the call-outs to the spec;  I think this is really helpful for all parties.